### ISSUE TITLE
Exporting parameters to file name

### ISSUE THREAD

**Issue by visjble** [LucianoCirino/efficiency-nodes-comfyui#101]
Hi Luciano,
Is there a way to export seed, filename, params, etc along with the save file node? Perhaps in .json?
Much appreciated.

**Comment by 4thanks** [comment]
just drag the output image to the comfyui, you will be surprised

**Comment by LucianoCirino** [comment]
@visjble does 4thanks's response answer your question?

**Comment by visjble** [comment]
I did not know about the parameters in the metadata, now I would have to deal with them, although very useful I did not want them there. I did however find the save_images function in the nodes, working with that, it's a long journey... If there is a custom module that allows to save params into the file name it would be molto meglio. Thanks.

**Comment by visjble** [comment]
I did not know about the parameters in the metadata, now I would have to deal with them, although very useful I did not want them there. I did however find the function where the image gets saved, working with that, it's a long journey... If there was a custom node that allows control over what params get to be as file name it would be molto meglio. Thanks. I do prefer ComfyUI to auto1111 ---EDIT: I just found SaveImgPrompt in the modules list, thanks
